src/features/metadata-instance-editor/__tests__/Instance.test.js

```diff
@@ -147,82 +148,149 @@ const userDefinedTemplateField = [
     },
 ];
 
-const intl = { formatMessage: () => {} };
+const intl = { formatMessage: jest.fn().mockReturnValue('Edit Metadata') };
```

Comment:
does this work: `const intl = { formatMessage: message => message.defaultMessage };`?

it would apply to additional message cases

Reply:
Changed to your suggestion.